frontend/templates/_footer.html

```diff
@@ -38,10 +38,10 @@
       <div class="col-sm-4">
         <ul class="list-unstyled">
           <li><strong>Information</strong></li>
-          <li><a href="http://legacy.pmg.org.za/rules-parliament-guide-procedure">Rules of Parliament</a></li>
+          <li><a href="/page/rules-parliament-guide-procedure">Rules of Parliament</a></li>
```

Comment:
The middle two links on this page are broken:

* National Council of Provinces  9th Edition: March 2008
* Joint Rules 6th Edition : June 2011